test-javascript.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
best to use `env` for portability and force exit on first error as in:

    #!/usr/bin/env bash
    set -e

Reply:
fixed in #9785d79